File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-1.java
class Solution {

    /*
    Similar to finding a target except we get a min
    Using 
    */
    public int findMin(int[] nums) {
        int left = 0;
        int right = nums.length -1;

        while(left < right){
            int mid = left + (right - left)/2;
            System.out.println(mid + " " + left + " " + right);
            if(nums[mid] > nums[right]){
                left = mid + 1;
            }
            else if(nums[mid] < nums[right]){
                right = mid;
            }
        }

        return nums[left];
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        numSet = {}
        for i in range(len(nums)):
            #print(numSet)
            if (target - nums[i]) in numSet:
                return [numSet[target - nums[i]],i]
            else:
                numSet[nums[i]] = i
        
    